### ISSUE TITLE
Enum generation error?

### ISSUE THREAD

**Issue by stevejbrother** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#29]
Hi Simon,

I have a public enum defined like this:

`public enum CatCode
    {
        [Description("Assembly")]
        Assemblies = 0,
        [Description("Part")]
        Parts = 1,
        [Description("Electrical")]
        Electricals = 2,
        [Description("Fittings")]
        Fixings = 3
    };`

In the template, I'm trying to add the enum to a column "CategoryCode":

EnumDefinitions.Add(new EnumDefinition {Schema = "dbo", Table = "Stock", Column = "CategoryCode", EnumType = "CatCode" });

But I'm getting this error message:
_Compiling transformation: The name 'EnumDefinitions' does not exist in the current context c:\Projects\Visual Studio 2013\MyDatabase.tt_

Am I missing something in the template, or heaven forbid, something is wrong with the template itself?

**Comment by ErikEJ** [comment]
Do you have an updated EF.Reverse.POCO.Core.ttinclude file in your project?

**Comment by sjh37** [comment]
I'll take a look

**Comment by sjh37** [comment]
All is fine with v2.19.1

``` c#
List<EnumDefinition> EnumsDefinitions = new List<EnumDefinition>();
```

is declared in EF.Reverse.POCO.Core.ttinclude line 89

Can you check what version you have in your database.tt and the EF.Reverse.POCO.Core.ttinclude files. It should be a comment at the top of the file.

**Comment by sjh37** [comment]
If you still think there is a problem, please let me know and I shall re-open the case.

**Comment by jxmiller** [comment]
I know this is old but... 
EnumDefinitions.Add...
vs
Enum**s**Definitions.Add...

**Comment by sjh37** [comment]
@jxmiller OMG I never spotted that before! Thanks.

**Comment by sjh37** [comment]
Released in v2.36.0

**Comment by jxmiller** [comment]
@sjh37 I wasn't even aware it was still an issue. Thanks for fixing.
